bootstrap.sh

```
@@ -1,10 +1,30 @@
 #!/bin/bash
+
+echo "Creating cluster..."
+kind create cluster --config cluster.yml
+kubectl wait --for=condition=Ready nodes --all --timeout=120s
+
+echo "Labeling nodes..."
+workers=( $(kubectl get nodes -o jsonpath='{range .items[*]}{.metadata.name}\n{end}' | grep kind-worker) )
+# Ensure at least one worker
+if [ ${#workers[@]} -lt 1 ]; then echo "no worker nodes found"; exit 1; fi
+# label first N as mysql
+for n in "${workers[@]:0:2}"; do kubectl label node "$n" app=mysql --overwrite; done
+# label remaining nodes as todoapp
+for n in "${workers[@]:2}"; do kubectl label node "$n" app=todoapp --overwrite; done
+
+echo "Tainting nodes..."
+kubectl get nodes -l app=mysql -o jsonpath='{.items[*].metadata.name}' | xargs -r -n1 -I {} kubectl taint nodes {} app=mysql:NoSchedule --overwrite
+echo "Launching mysql databases..."
 kubectl apply -f .infrastructure/mysql/ns.yml
 kubectl apply -f .infrastructure/mysql/configMap.yml
 kubectl apply -f .infrastructure/mysql/secret.yml
 kubectl apply -f .infrastructure/mysql/service.yml
 kubectl apply -f .infrastructure/mysql/statefulSet.yml
+kubectl -n <mysql> rollout status statefulset/mysql --timeout=120s
 
+
```

Before applying app manifests, add checks that the key manifest files exist (namespaces, statefulset, deployment). Currently the script blindly applies paths which may be missing and produce partial failures. Either fail early if essential files are missing or at least print which files are missing so the user can fix repository layout problems.